diff --git a/node_modules/@react-native-voice/voice/android/build.gradle b/node_modules/@react-native-voice/voice/android/build.gradle
index 8fce879..bf4af76 100644
--- a/node_modules/@react-native-voice/voice/android/build.gradle
+++ b/node_modules/@react-native-voice/voice/android/build.gradle
@@ -1,67 +1,48 @@
 apply plugin: 'com.android.library'
 
-repositories {
-    mavenLocal()
-    jcenter()
-    maven {
-        // For developing the library outside the context of the example app, expect `react-native`
-        // to be installed at `./node_modules`.
-        url "$projectDir/../node_modules/react-native/android"
-    }
-    maven {
-        // For developing the example app.
-        url "$projectDir/../../react-native/android"
-    }
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
-def DEFAULT_COMPILE_SDK_VERSION = 28
-def DEFAULT_BUILD_TOOLS_VERSION = "28.0.3"
-def DEFAULT_TARGET_SDK_VERSION = 28
-def DEFAULT_SUPPORT_LIB_VERSION = "28.0.0"
-
 android {
-    compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
-    buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
+    namespace "com.wenkesj.voice"
+    compileSdkVersion safeExtGet('compileSdkVersion', 34)
+    buildToolsVersion safeExtGet('buildToolsVersion', '34.0.0')
 
     defaultConfig {
-        minSdkVersion 15
-        targetSdkVersion rootProject.hasProperty('targetSdkVersion') ? rootProject.targetSdkVersion : DEFAULT_TARGET_SDK_VERSION
+        minSdkVersion safeExtGet('minSdkVersion', 21)
+        targetSdkVersion safeExtGet('targetSdkVersion', 34)
         versionCode 1
         versionName "1.0"
     }
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
-}
-
-buildscript {
-    if (project == rootProject) {
-        repositories {
-            jcenter()
-        }
-        dependencies {
-            classpath 'com.android.tools.build:gradle:3.3.2'
-
-            // NOTE: Do not place your application dependencies here; they belong
-            // in the individual module build.gradle files
-        }
+    lintOptions {
+        abortOnError false
     }
 }
 
-allprojects {
-    repositories {
-        jcenter()
+repositories {
+    google()
+    mavenCentral()
+    maven {
+        // For developing the library outside the context of the example app, expect `react-native`
+        // to be installed at `./node_modules`.
+        url "$projectDir/../node_modules/react-native/android"
+    }
+    maven {
+        // For developing the example app.
+        url "$projectDir/../../react-native/android"
     }
 }
 
-def supportVersion = rootProject.hasProperty('supportLibVersion') ? rootProject.supportLibVersion : DEFAULT_SUPPORT_LIB_VERSION
-
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     testImplementation 'junit:junit:4.12'
-    implementation "com.android.support:appcompat-v7:${supportVersion}"
+    implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.facebook.react:react-native:+'
 }
